### PR TITLE
Tb dialog update

### PR DESCRIPTION
**Pull request by TomasBlazek1**

### CODE REVIEW

File: src/JobPanel/ui/dialogs/edit_workspace_dialog.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+import os
+import sys
+
+from PyQt5 import QtWidgets
+from PyQt5 import QtCore
+
+from gm_base.geomop_analysis import Analysis
+
+class SetWorkspaceDialog():
```
Comment (jbrezmorf) [inline src/JobPanel/ui/dialogs/edit_workspace_dialog.py]: No reason why this has to be a class. A function suits better.

Comment (jbrezmorf) [reply src/JobPanel/ui/dialogs/edit_workspace_dialog.py]: Further if it is used on single place there is no need to place it into separate file.

Comment (jbrezmorf) [reply src/JobPanel/ui/dialogs/edit_workspace_dialog.py]: How it differs from WorkspaceSelectorWidget?

File: src/JobPanel/ui/main_window.py [hunk 2]
```diff
@@ -446,7 +446,14 @@ def _handle_stop_multijob_action(self):
...
     def _handle_options(self):
-        OptionsDialog(self, self.data,self.data.env_presets).show()
+        prewFileName = self.data.workspaces.get_path()
+        dialog = WorkspaceSelectorWidget(self, self.data.workspaces.get_path())
+        dialog.select_workspace()
+        if prewFileName != dialog.value:
+            self.data.reload_workspace(dialog.value)
+            if not Analysis.exists(self.data.workspaces.get_path(), self.data.config.analysis):
+                self.data.config.analysis = None
+            self.data.config.save()
```
Comment (radeksrb) [inline src/JobPanel/ui/main_window.py]: Tady by se asi mělo zavřít hlavni okno, jako se to dělo v OptionsDialog pomocí self.parent().close().

**Pull request merged**